test/sanity/validate-modules/ignore.txt

```diff
@@ -150,7 +150,6 @@ lib/ansible/modules/cloud/azure/azure_rm_acs.py E325
 lib/ansible/modules/cloud/azure/azure_rm_acs.py E326
 lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py E324
 lib/ansible/modules/cloud/azure/azure_rm_availabilityset_facts.py E322
```

Comment:
just curious, why some azure modules are ignored, others not?

Reply:
The ignore list was put in place to allow the tests to "pass" without having to fix all the pre-existing issues when the test was first added. As issues are fixed, they are removed from the list to prevent future regressions. The goal is for the ignore list to be empty.